### PR TITLE
fix(metrics): Rename misnamed aggregator option

### PR DESCRIPTION
**Pull request by untitaker**
#skip-changelog

### CODE REVIEW

**Review by flub — APPROVED** [review]

File: relay-metrics/src/aggregation.rs [hunk 1]
```diff
@@ -879,13 +879,13 @@ pub struct AggregatorConfig {
     /// Defaults to `None`, i.e. no limit.
     pub max_total_bucket_bytes: Option<usize>,
 
-    /// Maximum amount of bytes used for metrics aggregation per project.
+    /// Maximum amount of bytes used for metrics aggregation per project key.
     ///
     /// Similar measuring technique to `max_total_bucket_bytes`, but instead of a
     /// global/process-wide limit, it is enforced per project id.
```
Comment (jjbayer) [inline relay-metrics/src/aggregation.rs]:
```suggestion
    /// global/process-wide limit, it is enforced per project key.
```

Comment (jjbayer) [inline relay-metrics/src/aggregation.rs:885]:
```suggestion
    /// global/process-wide limit, it is enforced per project key.
```
```suggestion
    /// global/process-wide limit, it is enforced per project id.
```

**Review by jjbayer — APPROVED** [review]
One more comment on docs, plus some usages must be replaced by the new name, but other than that LGTM.

**Comment by jan-auer** [comment]
Note that test code still needs updating so this compiles.

**Comment by untitaker** [comment]
i forgot to push that. now it should work

**Pull request merged**